src/sentry/integrations/analytics.py

```diff
@@ -13,6 +13,16 @@ class IntegrationAddedEvent(analytics.Event):
     )
 
 
+class IntegrationDisabledNotified(analytics.Event):
+    type = "integration.disabled.notified"
+
+    attributes = (
+        analytics.Attribute("organization_id"),
+        analytics.Attribute("provider"),
```

Comment:
@chloeho7 I think we should still have some sort of integration/installation id field and then a type. That way we can do a join on the right table based on the type.